MiniApp/Classes/core/RealMiniApp.swift

```diff
@@ -47,41 +47,58 @@ internal class RealMiniApp {
+    func createMiniApp(appInfo: MiniAppInfo, manifest: MiniAppManifest? = nil, queryParams: String? = nil, completionHandler: @escaping (Result<MiniAppDisplayDelegate, Error>) -> Void, messageInterface: MiniAppMessageDelegate? = nil, adsDisplayer: MiniAppAdDisplayer? = nil) {
         getMiniApp(miniAppId: appInfo.id, miniAppVersion: appInfo.version.versionId) { (result) in
             switch result {
             case .success(let responseData):
                 if appInfo.version.versionId != responseData.version.versionId {
-                    self.downloadMiniApp(appInfo: responseData, queryParams: queryParams, completionHandler: completionHandler, messageInterface: messageInterface, adsDisplayer: adsDisplayer)
+                    self.downloadMiniApp(
+                            appInfo: responseData,
+                            manifest: manifest,
+                            queryParams: queryParams,
+                            completionHandler: completionHandler,
+                            messageInterface: messageInterface,
+                            adsDisplayer: adsDisplayer)
                     return
                 }
-                self.downloadMiniApp(appInfo: appInfo, queryParams: queryParams, completionHandler: completionHandler, messageInterface: messageInterface)
+                self.downloadMiniApp(
+                        appInfo: appInfo,
+                        manifest: manifest,
+                        queryParams: queryParams,
+                        completionHandler: completionHandler,
+                        messageInterface: messageInterface)
             case .failure(let error):
-                self.handleMiniAppDownloadError(appId: appInfo.id,
-                                 error: error,
-                                 queryParams: queryParams,
-                                 completionHandler: completionHandler,
-                                 messageInterface: messageInterface,
-                                 adsDisplayer: adsDisplayer)
+                self.handleMiniAppDownloadError(
+                        appId: appInfo.id,
+                        manifest: manifest,
+                        error: error,
+                        queryParams: queryParams,
+                        completionHandler: completionHandler,
+                        messageInterface: messageInterface,
+                        adsDisplayer: adsDisplayer)
             } }
     }
 
     func createMiniApp(appId: String, version: String? = nil, queryParams: String? = nil, completionHandler: @escaping (Result<MiniAppDisplayDelegate, MASDKError>) -> Void, messageInterface: MiniAppMessageDelegate? = nil, adsDisplayer: MiniAppAdDisplayer? = nil) {
         getMiniApp(miniAppId: appId, miniAppVersion: version) { (result) in
+            let manifest = self.miniAppManifestStorage.getManifestInfo(forMiniApp: appId)?.miniAppManifest
```

Comment:
When the mini-app is downloaded for the first time, this manifest object would be nil. Also, I see you are passing this `manifest` object all the way to `func getMiniAppView()` method.

In that case, you could retrieve the manifest here -> https://github.com/rakutentech/ios-miniapp/pull/228/files#diff-af0f0dfc2aba552e5313b2588ffed2ac98cffd616f7caca2105ac7660572fc6cR167

Doing so, it will give you the latest manifest and also you can avoid passing `manifest` object